src/main/groovy/uk/gov/hmrc/jenkinsjobbuilders/domain/authorisation/Permission.groovy

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmrc.jenkinsjobbuilders.domain.authorisation
+
+import uk.gov.hmrc.jenkinsjobbuilders.domain.Setting
+
+class Permission implements Setting {
+
+    final String permissionIdentifier
```

Comment:
I wonder if it's worth making this an Enum, it might make it easier for folks to know what the options are. It would also ensure that people couldn't specify a permission that we didn't want them to have.